plugins/inputs/aerospike/aerospike.go

```diff
@@ -228,8 +230,8 @@ func (a *Aerospike) gatherServer(acc telegraf.Accumulator, hostPort string) erro
 	return nil
 }
 
-func (a *Aerospike) getNodeInfo(n *as.Node) (map[string]string, error) {
-	stats, err := as.RequestNodeStats(n)
+func (a *Aerospike) getNodeInfo(n *as.Node, infoPolicy *as.InfoPolicy) (map[string]string, error) {
+	stats, err := n.RequestInfo(infoPolicy)
```

Comment:
Hi, I know it's a little late, but I think this has introduced a bug. Shouldn't this be added to this method call?
```suggestion
	stats, err := n.RequestInfo(infoPolicy, "statistics")
```

Reply:
@Hipska Thanks, Let me quickly test it out and create a PR